supplier_app/__init__.py

```diff
@@ -3,13 +3,18 @@
 
 TAXPAYER_BANK_ACCOUNT_ALLOWED_FILE_EXTENSIONS = ['.pdf']
 
-TAXPAYER_STATUS = [
-    ("ACTIVE", "Active"),
-    ("CHANGE REQUIRED", "Change required"),
-    ("PENDING", "Pending"),
-    ("REFUSED", "Refused"),
-]
+TAXPAYER_STATUS = {
+    "ACTIVE": "Active",
+    "CHANGE REQUIRED": "Change required",
+    "PENDING": "Pending",
+    "DENIED": "Denied",
+}
+
 TAXPAYER_STATUS_AP = [
-    'CHANGE REQUIRED',
-    'PENDING',
+    'Change required',
+    'Pending',
 ]
```

Comment:
What is the difference between TAXPAYER_STATUS and TAXPAYER_STATUS_AP?

Reply:
TAXPAYER_STATUS_AP is a list used to show in the AP home the taxpayers which have any of those status. And TAXPAYER_STATUS is used to control every status of a taxpayer. BTW I think it can be just one dictionary

Reply:
Yes, maybe you can find out a simpler way!